src/core/link.js

```diff
@@ -90,6 +90,14 @@ Link.prototype = {
 			attrs
 		);
 
+		if(!linkAttrs.target) {
+			const buttonCfg = this._editor.config.buttonCfg || {};
+			const defaultLinkTarget = buttonCfg['linkEdit'] && buttonCfg['linkEdit']['defaultLinkTarget'];
+			if(defaultLinkTarget) {
+				linkAttrs.target = defaultLinkTarget;
+			}
+		}
```

Comment:
`CKEDITOR.Link` is a low-level utility, for creating and handling links. `buttonCfg` is not an API intended to affect links in general, or link utilities. If is an API to configure specific button components. `linkEdit` is the key identifying [ButtonLinkEdit component](https://github.com/liferay/alloy-editor/blob/master/src/components/buttons/button-link-edit.jsx#L60). We should not couple `linkEdit` with links in general, like in this solution.

I could see this maybe as a new configuration option, on the level of `buttonCfg`. Something like
```js
linkCfg: {
    defaultTarget: '_blank'
}
```
Then, we could read it and merge with other attributes, in `link.js`.

Reply:
@markocikos thanks for the proposal, i changed it to linkCfg.defaultTarget - makes total sense!